flank-scripts/src/main/kotlin/flank/scripts/shell/ops/BuildGameLoopExampleCommand.kt

```diff
@@ -0,0 +1,35 @@
+package flank.scripts.shell.ops
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import flank.scripts.shell.utils.failIfWindows
+import flank.scripts.shell.utils.iOSTestProjectsPath
+import java.nio.file.Paths
+
+object BuildGameLoopExampleCommand : CliktCommand(
+    name = "build_ios_gameloop_example",
+    help = "Build ios game loop example app"
+) {
```

Comment:
please update documentation in README.md of flank-scripts

Reply:
Updated